utilities/javascript/binarySearch.js

```diff
@@ -12,15 +12,15 @@ function binarySearch(arr, target) {
             right = mid - 1;
         }
     }
-    return left;
+    return arr[left] === target ? left : -1;
```

Comment:
Actually we don't have to check for the value here. Because if the value was the target, it would have been returned in `return mid`. We can simply `return -1` here.

Reply:
If we don't check then `binarySearch([1, 2, 3, 10], 1)` would return `-1` because `left === right`